unime/src/lib/components/StatusIndicator.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition';
+
+  import type { ValidationStatus } from '@bindings/user_prompt/ValidationStatus';
+  import { createPopover, melt } from '@melt-ui/svelte';
+
+  import { Image } from '$lib/components';
+  import { hash } from '$lib/utils';
+
+  import Check from '~icons/ph/check-bold';
+  import QuestionMark from '~icons/ph/question-mark-bold';
+  import X from '~icons/ph/x-bold';
```

Comment:
Replace icon imports with

`import {CheckBoldIcon, QUestionMarkBoldIcon, XBoldIcon} from '$lib/icons';`

This was part of a refactoring done while the PR was pending.